src/components/Card.js

```diff
@@ -18,23 +29,36 @@ export default class Card {
 
   _setEventListeners() {
     this._likeButton.addEventListener("click", () => {
-      this._handleLikeIcon();
+      this._handleLikeClick(this);
     });
 
     this._deleteButton.addEventListener("click", () => {
-      this._handleDeleteCard();
+      this._handleDeleteSubmit(this._id, this);
     });
 
     this._cardImage.addEventListener("click", () => {
       this._handleImageClick({ name: this._name, link: this._link });
     });
   }
 
-  _handleLikeIcon() {
-    this._likeButton.classList.toggle("card__like-button_active");
+  _isLiked() {
```

Comment:
`  _isLiked() {
`

Please, delete this method because it's reassigned in line 13. It becomes just a **boolean**